README.md

```diff
@@ -230,14 +209,43 @@ TBD
 
         $ poetry install
 
-### Run tests:
+### Run tests
 
         $ poetry run pytest
 
 ### Run tests when source changes (uses [pytest-watcher](https://github.com/olzhasar/pytest-watcher)):
 
         $ poetry run ptw . --now --clear
 
+### Releasing
+
+* Update the CHANGELOG to reflect the latest version and the
+  changes included in the release.
+
+* Show the current version and the possible next versions:
+
+        $ bump-my-version show-bump
+        0.3.0 ── bump ─┬─ major ─ 1.0.0
+                       ├─ minor ─ 0.4.0
+                       ╰─ patch ─ 0.3.1
+
+* Bump the version to the desired number, for example:
+
+        $ bump-my-version bump minor
```

Comment:
If I'm reading this right, you'll need to know the version you're bumping to before you update the CHANGELOG (unless there's a `bump` option to automatically replace some placeholder text with the actual version in the CHANGELOG itself). In that case, lines 222 and 223 would be more logical if moved after line 230.

Reply:
Fixed this. Putting UNRELEASED in the CHANGELOG will get replaced by the new version.